appengine/datastore/app.rb

```diff
@@ -28,7 +28,7 @@
   datastore.save visit
 
   # Query the last 10 visits from the Datastore
-  query     = datastore.query("Visit").order("timestamp", :desc).limit(10)
+  query     = datastore.query("Visit").order("timestamp", :desc).limit 10
```

Comment:
Is rubocop really insisting on removing those parens?

Reply:
Yup

Reply:
ah, `Style/MethodCallWithArgsParentheses`, my old nemesis. Enabling `AllowParenthesesInChaining` will permit this specific case, which I think is preferable. Mixing parenthesis style on the same line is kinda gross.